robottelo/constants.py

```diff
@@ -258,9 +258,14 @@
 DEFAULT_SUBSCRIPTION_NAME = (
     'Red Hat Enterprise Linux Server Entry Level, Self-support')
 
-LANGUAGES = ["en", "en_GB",
-             "fr", "gl",
-             "ja", "sv_SE"]
+LANGUAGES = [
+    "zh_TW",
+    "Japanese",
+    "Deutsch",
+    "it",
+    "ko",
+    "sv_SE",
+]
```

Comment:
As it is almost no difference what language to test from QA point of view here (at least some of them should be selected)
And honestly, I don't gave enough time to investigate seriously (few solutions that I tried didn't help) and find proper solution for next issue:
http://stackoverflow.com/questions/22168651/how-to-enter-left-parentheses-into-a-text-box
http://stackoverflow.com/questions/19704559/selenium-sendkeys-not-working-for-open-brackets-and-harsh-keys-when-using-java